website/source/docs/enterprise/index.html.md

```diff
@@ -18,6 +18,7 @@ increases both scalability and resilience. Features include:
 - [Redundancy Zones](/docs/enterprise/redundancy/index.html)
 - [Advanced Federation for Complex Network
   Topologies](/docs/enterprise/federation/index.html)
+- [Network Segments](/docs/guides/segments.html)
```

Comment:
ping @KFishner this is fine for now but do you want to write a marketing landing thing for later?

Reply:
https://github.com/hashicorp/consul/pull/3441